### PR TITLE
[test](executor)add crud regression test for resource group

### PR DESCRIPTION
**Pull request by wangbo**
# Proposed changes

add crud regression test for resource group

Describe your changes.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by wangbo** [comment]
run buildall

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 32.69 seconds
 stream load tsv:          424 seconds loaded 74807831229 Bytes, about 168 MB/s
 stream load json:         23 seconds loaded 2358488459 Bytes, about 97 MB/s
 stream load orc:          60 seconds loaded 1101869774 Bytes, about 17 MB/s
 stream load parquet:          30 seconds loaded 861443392 Bytes, about 27 MB/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230516084834_clickbench_pr_144452.html

File: regression-test/suites/resource_group_p0/test_resource_group_crud.groovy [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    }
+    assertTrue(rgNameTvfResult);
+    if (!item_tvf.equals("cpu_share")) {
+        itemTvfResult = false
+    }
+    assertTrue(itemTvfResult)
+    if (!value_tvf.equals("1")) {
+        valueTvfResult = false;
+    }
+    assertTrue(valueTvfResult)
```
Comment (luozenglin) [inline regression-test/suites/resource_group_p0/test_resource_group_crud.groovy:117]: maybe it's cleaner to use qt_test:
```
qt_select """select Name, Item, Value from resource_groups() where name='${rg_name}' order by 1, 2, 3;"""
```

**Comment by wangbo** [comment]
ru build p0

**Comment by wangbo** [comment]
run buildall

**Review by luozenglin — APPROVED** [review]
LGTM

**Comment by wangbo** [comment]
run buildall

**Comment by wangbo** [comment]
run buildall

**Pull request merged**